### ISSUE TITLE
Tensorflow breaks entirely on the following input

### ISSUE THREAD

**Issue by Perendinaire** [tensorflow/playground#3]
When inputting the following parameters, both the neurons and the output of tensorflow breaks entirely, giving NaN. Tested across the latest Firefox 47, Chrome 49 and Opera 37 browsers.

http://playground.tensorflow.org/#activation=tanh&batchSize=1&dataset=spiral&regDataset=reg-plane&learningRate=1&regularizationRate=0&noise=50&networkShape=8&seed=0.42182&showTestData=false&discretize=false&percTrainData=50&x=true&y=true&xTimesY=false&xSquared=false&ySquared=false&cosX=false&sinX=false&cosY=false&sinY=false&collectStats=false&problem=regression

**Comment by IMG-PRCSNG** [comment]
Looks like the learning rate is too high. LR < 0.3 seems to work.

**Comment by dsmilkov** [comment]
Yes. This is the expected behavior. Whenever you get a NaN, we show it using a grey color.

In this case, you are using regression as a problem type, and the output of the last neuron is just a linear function (unbounded output, so the error derivative is unbounded). So, since your learning rate is too high, the network is going to escape the stable region and diverge.
